Watching for file changes with StatReloader
This is the scrape input form
Starting new HTTPS connection (1): my-site1-html-ipfs.vercel.app:443
https://my-site1-html-ipfs.vercel.app:443 "GET / HTTP/1.1" 200 None
This is the scrape input form
This is the scrape input form
Starting new HTTPS connection (1): www.coindesk.com:443
https://www.coindesk.com:443 "GET /business/2023/06/22/opnx-files-defamation-lawsuit-against-mike-dudas-issues-justice-tokens/?utm_medium=social&utm_term=organic&utm_campaign=coindesk_main&utm_content=editorial&utm_source=twitter HTTP/1.1" 200 None
Internal Server Error: /scrape/
Traceback (most recent call last):
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
psycopg2.errors.StringDataRightTruncation: value too long for type character varying(200)


The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/src/django/contents/views.py", line 82, in scrape
    page.save()
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/base.py", line 814, in save
    self.save_base(
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/base.py", line 877, in save_base
    updated = self._save_table(
              ^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/base.py", line 1020, in _save_table
    results = self._do_insert(
              ^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/base.py", line 1061, in _do_insert
    return manager._insert(
           ^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/query.py", line 1805, in _insert
    return query.get_compiler(using=using).execute_sql(returning_fields)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/sql/compiler.py", line 1822, in execute_sql
    cursor.execute(sql, params)
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 67, in execute
    return self._execute_with_wrappers(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 80, in _execute_with_wrappers
    return executor(sql, params, many, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 84, in _execute
    with self.db.wrap_database_errors:
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
django.db.utils.DataError: value too long for type character varying(200)

This is the scrape input form
This is the scrape input form
Starting new HTTPS connection (1): www.coindesk.com:443
https://www.coindesk.com:443 "GET /business/2023/06/22/opnx-files-defamation-lawsuit-against-mike-dudas-issues-justice-tokens/?utm_medium=social&utm_term=organic&utm_campaign=coindesk_main&utm_content=editorial&utm_source=twitter HTTP/1.1" 200 None
Internal Server Error: /scrape/
Traceback (most recent call last):
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
psycopg2.errors.StringDataRightTruncation: value too long for type character varying(200)


The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/src/django/contents/views.py", line 82, in scrape
    page.save()
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/base.py", line 814, in save
    self.save_base(
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/base.py", line 877, in save_base
    updated = self._save_table(
              ^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/base.py", line 1020, in _save_table
    results = self._do_insert(
              ^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/base.py", line 1061, in _do_insert
    return manager._insert(
           ^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/query.py", line 1805, in _insert
    return query.get_compiler(using=using).execute_sql(returning_fields)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/sql/compiler.py", line 1822, in execute_sql
    cursor.execute(sql, params)
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 67, in execute
    return self._execute_with_wrappers(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 80, in _execute_with_wrappers
    return executor(sql, params, many, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 84, in _execute
    with self.db.wrap_database_errors:
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
django.db.utils.DataError: value too long for type character varying(200)

/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/src/django/vercel_app/settings.py changed, reloading.
Watching for file changes with StatReloader
This is the scrape input form
Starting new HTTPS connection (1): www.coindesk.com:443
https://www.coindesk.com:443 "GET /business/2023/06/22/opnx-files-defamation-lawsuit-against-mike-dudas-issues-justice-tokens/?utm_medium=social&utm_term=organic&utm_campaign=coindesk_main&utm_content=editorial&utm_source=twitter HTTP/1.1" 200 None
Internal Server Error: /scrape/
Traceback (most recent call last):
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
psycopg2.errors.StringDataRightTruncation: value too long for type character varying(200)


The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/src/django/contents/views.py", line 82, in scrape
    page.save()
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/base.py", line 814, in save
    self.save_base(
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/base.py", line 877, in save_base
    updated = self._save_table(
              ^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/base.py", line 1020, in _save_table
    results = self._do_insert(
              ^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/base.py", line 1061, in _do_insert
    return manager._insert(
           ^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/query.py", line 1805, in _insert
    return query.get_compiler(using=using).execute_sql(returning_fields)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/sql/compiler.py", line 1822, in execute_sql
    cursor.execute(sql, params)
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 102, in execute
    return super().execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 67, in execute
    return self._execute_with_wrappers(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 80, in _execute_with_wrappers
    return executor(sql, params, many, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 84, in _execute
    with self.db.wrap_database_errors:
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
django.db.utils.DataError: value too long for type character varying(200)

/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/src/django/contents/models.py changed, reloading.
Watching for file changes with StatReloader
This is the scrape input form
Starting new HTTPS connection (1): www.coindesk.com:443
https://www.coindesk.com:443 "GET /business/2023/06/22/opnx-files-defamation-lawsuit-against-mike-dudas-issues-justice-tokens/?utm_medium=social&utm_term=organic&utm_campaign=coindesk_main&utm_content=editorial&utm_source=twitter HTTP/1.1" 200 None
Internal Server Error: /scrape/
Traceback (most recent call last):
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
psycopg2.errors.StringDataRightTruncation: value too long for type character varying(200)


The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/src/django/contents/views.py", line 82, in scrape
    page.save()
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/base.py", line 814, in save
    self.save_base(
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/base.py", line 877, in save_base
    updated = self._save_table(
              ^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/base.py", line 1020, in _save_table
    results = self._do_insert(
              ^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/base.py", line 1061, in _do_insert
    return manager._insert(
           ^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/query.py", line 1805, in _insert
    return query.get_compiler(using=using).execute_sql(returning_fields)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/sql/compiler.py", line 1822, in execute_sql
    cursor.execute(sql, params)
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 102, in execute
    return super().execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 67, in execute
    return self._execute_with_wrappers(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 80, in _execute_with_wrappers
    return executor(sql, params, many, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 84, in _execute
    with self.db.wrap_database_errors:
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
django.db.utils.DataError: value too long for type character varying(200)

This is the scrape input form
This is the scrape input form
Starting new HTTPS connection (1): www.coindesk.com:443
https://www.coindesk.com:443 "GET /business/2023/06/22/opnx-files-defamation-lawsuit-against-mike-dudas-issues-justice-tokens/?utm_medium=social&utm_term=organic&utm_campaign=coindesk_main&utm_content=editorial&utm_source=twitter HTTP/1.1" 200 None
Internal Server Error: /scrape/
Traceback (most recent call last):
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
psycopg2.errors.StringDataRightTruncation: value too long for type character varying(200)


The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/src/django/contents/views.py", line 82, in scrape
    page.save()
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/base.py", line 814, in save
    self.save_base(
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/base.py", line 877, in save_base
    updated = self._save_table(
              ^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/base.py", line 1020, in _save_table
    results = self._do_insert(
              ^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/base.py", line 1061, in _do_insert
    return manager._insert(
           ^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/query.py", line 1805, in _insert
    return query.get_compiler(using=using).execute_sql(returning_fields)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/sql/compiler.py", line 1822, in execute_sql
    cursor.execute(sql, params)
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 102, in execute
    return super().execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 67, in execute
    return self._execute_with_wrappers(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 80, in _execute_with_wrappers
    return executor(sql, params, many, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 84, in _execute
    with self.db.wrap_database_errors:
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
django.db.utils.DataError: value too long for type character varying(200)

This is the scrape input form
This is the scrape input form
Starting new HTTPS connection (1): stackoverflow.com:443
https://stackoverflow.com:443 "GET /questions/67031624/how-to-debug-or-handle-vercel-function-invocation-failed-error HTTP/1.1" 200 None
This is the scrape input form
This is the scrape input form
Starting new HTTPS connection (1): stackoverflow.com:443
https://stackoverflow.com:443 "GET /questions/67031624/how-to-debug-or-handle-vercel-function-invocation-failed-error HTTP/1.1" 200 None
This is the scrape input form
This is the scrape input form
Starting new HTTPS connection (1): www.coindesk.com:443
https://www.coindesk.com:443 "GET /business/2023/06/22/opnx-files-defamation-lawsuit-against-mike-dudas-issues-justice-tokens/?utm_medium=social&utm_term=organic&utm_campaign=coindesk_main&utm_content=editorial&utm_source=twitter HTTP/1.1" 200 None
Internal Server Error: /scrape/
Traceback (most recent call last):
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
psycopg2.errors.StringDataRightTruncation: value too long for type character varying(200)


The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/src/django/contents/views.py", line 82, in scrape
    page.save()
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/base.py", line 814, in save
    self.save_base(
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/base.py", line 877, in save_base
    updated = self._save_table(
              ^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/base.py", line 1020, in _save_table
    results = self._do_insert(
              ^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/base.py", line 1061, in _do_insert
    return manager._insert(
           ^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/query.py", line 1805, in _insert
    return query.get_compiler(using=using).execute_sql(returning_fields)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/sql/compiler.py", line 1822, in execute_sql
    cursor.execute(sql, params)
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 102, in execute
    return super().execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 67, in execute
    return self._execute_with_wrappers(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 80, in _execute_with_wrappers
    return executor(sql, params, many, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 84, in _execute
    with self.db.wrap_database_errors:
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
django.db.utils.DataError: value too long for type character varying(200)

/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/src/django/contents/models.py changed, reloading.
Watching for file changes with StatReloader
/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/src/django/contents/models.py changed, reloading.
Watching for file changes with StatReloader
This is the scrape input form
/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/src/django/contents/models.py changed, reloading.
Watching for file changes with StatReloader
This is the scrape input form
Starting new HTTPS connection (1): www.coindesk.com:443
https://www.coindesk.com:443 "GET /business/2023/06/22/opnx-files-defamation-lawsuit-against-mike-dudas-issues-justice-tokens/?utm_medium=social&utm_term=organic&utm_campaign=coindesk_main&utm_content=editorial&utm_source=twitter HTTP/1.1" 200 None
Internal Server Error: /scrape/
Traceback (most recent call last):
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
psycopg2.errors.StringDataRightTruncation: value too long for type character varying(200)


The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/src/django/contents/views.py", line 82, in scrape
    page.save()
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/base.py", line 814, in save
    self.save_base(
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/base.py", line 877, in save_base
    updated = self._save_table(
              ^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/base.py", line 1020, in _save_table
    results = self._do_insert(
              ^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/base.py", line 1061, in _do_insert
    return manager._insert(
           ^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/query.py", line 1805, in _insert
    return query.get_compiler(using=using).execute_sql(returning_fields)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/sql/compiler.py", line 1822, in execute_sql
    cursor.execute(sql, params)
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 102, in execute
    return super().execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 67, in execute
    return self._execute_with_wrappers(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 80, in _execute_with_wrappers
    return executor(sql, params, many, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 84, in _execute
    with self.db.wrap_database_errors:
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
django.db.utils.DataError: value too long for type character varying(200)

This is the scrape input form
This is the scrape input form
This is the scrape input form
This is the scrape input form
Starting new HTTPS connection (1): www.coindesk.com:443
https://www.coindesk.com:443 "GET /business/2023/06/22/opnx-files-defamation-lawsuit-against-mike-dudas-issues-justice-tokens/?utm_medium=social&utm_term=organic&utm_campaign=coindesk_main&utm_content=editorial&utm_source=twitter HTTP/1.1" 200 None
Internal Server Error: /scrape/
Traceback (most recent call last):
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
psycopg2.errors.StringDataRightTruncation: value too long for type character varying(200)


The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/src/django/contents/views.py", line 82, in scrape
    page.save()
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/base.py", line 814, in save
    self.save_base(
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/base.py", line 877, in save_base
    updated = self._save_table(
              ^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/base.py", line 1020, in _save_table
    results = self._do_insert(
              ^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/base.py", line 1061, in _do_insert
    return manager._insert(
           ^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/query.py", line 1805, in _insert
    return query.get_compiler(using=using).execute_sql(returning_fields)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/sql/compiler.py", line 1822, in execute_sql
    cursor.execute(sql, params)
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 102, in execute
    return super().execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 67, in execute
    return self._execute_with_wrappers(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 80, in _execute_with_wrappers
    return executor(sql, params, many, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 84, in _execute
    with self.db.wrap_database_errors:
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
django.db.utils.DataError: value too long for type character varying(200)

This is the scrape input form
This is the scrape input form
Starting new HTTPS connection (1): www.youtube.com:443
https://www.youtube.com:443 "GET /watch?v=HEV1PWycOuQ HTTP/1.1" 200 None
This is the scrape input form
Starting new HTTPS connection (1): www.youtube.com:443
https://www.youtube.com:443 "GET /watch?v=HEV1PWycOuQ HTTP/1.1" 200 None
This is the scrape input form
This is the scrape input form
Starting new HTTPS connection (1): www.coindesk.com:443
https://www.coindesk.com:443 "GET /business/2023/06/22/opnx-files-defamation-lawsuit-against-mike-dudas-issues-justice-tokens/?utm_medium=social&utm_term=organic&utm_campaign=coindesk_main&utm_content=editorial&utm_source=twitter HTTP/1.1" 200 None
Internal Server Error: /scrape/
Traceback (most recent call last):
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
psycopg2.errors.StringDataRightTruncation: value too long for type character varying(200)


The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/src/django/contents/views.py", line 82, in scrape
    page.save()
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/base.py", line 814, in save
    self.save_base(
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/base.py", line 877, in save_base
    updated = self._save_table(
              ^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/base.py", line 1020, in _save_table
    results = self._do_insert(
              ^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/base.py", line 1061, in _do_insert
    return manager._insert(
           ^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/query.py", line 1805, in _insert
    return query.get_compiler(using=using).execute_sql(returning_fields)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/sql/compiler.py", line 1822, in execute_sql
    cursor.execute(sql, params)
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 102, in execute
    return super().execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 67, in execute
    return self._execute_with_wrappers(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 80, in _execute_with_wrappers
    return executor(sql, params, many, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 84, in _execute
    with self.db.wrap_database_errors:
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
django.db.utils.DataError: value too long for type character varying(200)

This is the scrape input form
This is the scrape input form
This is the scrape input form
Starting new HTTPS connection (1): www.coindesk.com:443
https://www.coindesk.com:443 "GET /business/2023/06/22/opnx-files-defamation-lawsuit-against-mike-dudas-issues-justice-tokens/?utm_medium=social&utm_term=organic&utm_campaign=coindesk_main&utm_content=editorial&utm_source=twitter HTTP/1.1" 200 None
Internal Server Error: /scrape/
Traceback (most recent call last):
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
psycopg2.errors.StringDataRightTruncation: value too long for type character varying(200)


The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/src/django/contents/views.py", line 82, in scrape
    page.save()
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/base.py", line 814, in save
    self.save_base(
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/base.py", line 877, in save_base
    updated = self._save_table(
              ^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/base.py", line 1020, in _save_table
    results = self._do_insert(
              ^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/base.py", line 1061, in _do_insert
    return manager._insert(
           ^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/query.py", line 1805, in _insert
    return query.get_compiler(using=using).execute_sql(returning_fields)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/sql/compiler.py", line 1822, in execute_sql
    cursor.execute(sql, params)
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 102, in execute
    return super().execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 67, in execute
    return self._execute_with_wrappers(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 80, in _execute_with_wrappers
    return executor(sql, params, many, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 84, in _execute
    with self.db.wrap_database_errors:
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
django.db.utils.DataError: value too long for type character varying(200)

Not Found: /favicon.ico
This is the scrape input form
This is the scrape input form
Starting new HTTPS connection (1): www.youtube.com:443
https://www.youtube.com:443 "GET /watch?v=HEV1PWycOuQ HTTP/1.1" 200 None
/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/src/django/contents/forms.py changed, reloading.
Watching for file changes with StatReloader
/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/src/django/contents/forms.py changed, reloading.
Watching for file changes with StatReloader
This is the scrape input form
This is the scrape input form
Starting new HTTPS connection (1): www.youtube.com:443
https://www.youtube.com:443 "GET /watch?v=HEV1PWycOuQ HTTP/1.1" 200 None
This is the scrape input form
This is the scrape input form
Starting new HTTPS connection (1): www.coindesk.com:443
https://www.coindesk.com:443 "GET /business/2023/06/22/opnx-files-defamation-lawsuit-against-mike-dudas-issues-justice-tokens/?utm_medium=social&utm_term=organic&utm_campaign=coindesk_main&utm_content=editorial&utm_source=twitter HTTP/1.1" 200 None
Internal Server Error: /scrape/
Traceback (most recent call last):
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
psycopg2.errors.StringDataRightTruncation: value too long for type character varying(200)


The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/src/django/contents/views.py", line 82, in scrape
    page.save()
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/base.py", line 814, in save
    self.save_base(
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/base.py", line 877, in save_base
    updated = self._save_table(
              ^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/base.py", line 1020, in _save_table
    results = self._do_insert(
              ^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/base.py", line 1061, in _do_insert
    return manager._insert(
           ^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/query.py", line 1805, in _insert
    return query.get_compiler(using=using).execute_sql(returning_fields)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/sql/compiler.py", line 1822, in execute_sql
    cursor.execute(sql, params)
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 102, in execute
    return super().execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 67, in execute
    return self._execute_with_wrappers(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 80, in _execute_with_wrappers
    return executor(sql, params, many, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 84, in _execute
    with self.db.wrap_database_errors:
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
django.db.utils.DataError: value too long for type character varying(200)

This is the scrape input form
This is the scrape input form
Starting new HTTPS connection (1): www.coindesk.com:443
https://www.coindesk.com:443 "GET /business/2023/06/22/opnx-files-defamation-lawsuit-against-mike-dudas-issues-justice-tokens/?utm_medium=social&utm_term=organic&utm_campaign=coindesk_main&utm_content=editorial&utm_source=twitter HTTP/1.1" 200 None
Internal Server Error: /scrape/
Traceback (most recent call last):
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
psycopg2.errors.StringDataRightTruncation: value too long for type character varying(200)


The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/src/django/contents/views.py", line 82, in scrape
    page.save()
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/base.py", line 814, in save
    self.save_base(
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/base.py", line 877, in save_base
    updated = self._save_table(
              ^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/base.py", line 1020, in _save_table
    results = self._do_insert(
              ^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/base.py", line 1061, in _do_insert
    return manager._insert(
           ^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/query.py", line 1805, in _insert
    return query.get_compiler(using=using).execute_sql(returning_fields)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/sql/compiler.py", line 1822, in execute_sql
    cursor.execute(sql, params)
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 102, in execute
    return super().execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 67, in execute
    return self._execute_with_wrappers(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 80, in _execute_with_wrappers
    return executor(sql, params, many, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 84, in _execute
    with self.db.wrap_database_errors:
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
django.db.utils.DataError: value too long for type character varying(200)

/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/src/django/contents/migrations/0001_initial.py changed, reloading.
Watching for file changes with StatReloader
/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/src/django/contents/admin.py changed, reloading.
Watching for file changes with StatReloader
/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/src/django/contents/models.py changed, reloading.
Watching for file changes with StatReloader
/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/src/django/contents/urls.py changed, reloading.
Watching for file changes with StatReloader
/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/src/django/contents/views.py changed, reloading.
Watching for file changes with StatReloader
This is the scrape input form
Starting new HTTPS connection (1): www.coindesk.com:443
https://www.coindesk.com:443 "GET /business/2023/06/22/opnx-files-defamation-lawsuit-against-mike-dudas-issues-justice-tokens/?utm_medium=social&utm_term=organic&utm_campaign=coindesk_main&utm_content=editorial&utm_source=twitter HTTP/1.1" 200 None
Internal Server Error: /scrape/
Traceback (most recent call last):
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
psycopg2.errors.StringDataRightTruncation: value too long for type character varying(200)


The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/src/django/contents/views.py", line 77, in scrape
    page.save()
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/base.py", line 814, in save
    self.save_base(
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/base.py", line 877, in save_base
    updated = self._save_table(
              ^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/base.py", line 1020, in _save_table
    results = self._do_insert(
              ^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/base.py", line 1061, in _do_insert
    return manager._insert(
           ^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/query.py", line 1805, in _insert
    return query.get_compiler(using=using).execute_sql(returning_fields)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/sql/compiler.py", line 1822, in execute_sql
    cursor.execute(sql, params)
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 102, in execute
    return super().execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 67, in execute
    return self._execute_with_wrappers(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 80, in _execute_with_wrappers
    return executor(sql, params, many, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 84, in _execute
    with self.db.wrap_database_errors:
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
django.db.utils.DataError: value too long for type character varying(200)

Not Found: /favicon.ico
This is the scrape input form
This is the scrape input form
Starting new HTTPS connection (1): www.coindesk.com:443
https://www.coindesk.com:443 "GET /business/2023/06/22/opnx-files-defamation-lawsuit-against-mike-dudas-issues-justice-tokens/?utm_medium=social&utm_term=organic&utm_campaign=coindesk_main&utm_content=editorial&utm_source=twitter HTTP/1.1" 200 None
Internal Server Error: /scrape/
Traceback (most recent call last):
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
psycopg2.errors.StringDataRightTruncation: value too long for type character varying(200)


The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/src/django/contents/views.py", line 77, in scrape
    page.save()
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/base.py", line 814, in save
    self.save_base(
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/base.py", line 877, in save_base
    updated = self._save_table(
              ^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/base.py", line 1020, in _save_table
    results = self._do_insert(
              ^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/base.py", line 1061, in _do_insert
    return manager._insert(
           ^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/query.py", line 1805, in _insert
    return query.get_compiler(using=using).execute_sql(returning_fields)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/models/sql/compiler.py", line 1822, in execute_sql
    cursor.execute(sql, params)
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 102, in execute
    return super().execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 67, in execute
    return self._execute_with_wrappers(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 80, in _execute_with_wrappers
    return executor(sql, params, many, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 84, in _execute
    with self.db.wrap_database_errors:
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/.venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
django.db.utils.DataError: value too long for type character varying(200)

This is the scrape input form
/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/src/django/contents/models.py changed, reloading.
Watching for file changes with StatReloader
/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/src/django/contents/forms.py changed, reloading.
Watching for file changes with StatReloader
This is the scrape input form
Starting new HTTPS connection (1): www.youtube.com:443
https://www.youtube.com:443 "GET /watch?v=HEV1PWycOuQ HTTP/1.1" 200 None
This is the scrape input form
/Users/cooperreed/Documents/GitHub/BMX-bookmark-extractor/src/django/contents/forms.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
This is the scrape input form
This is the scrape input form
Starting new HTTPS connection (1): www.youtube.com:443
https://www.youtube.com:443 "GET /watch?v=HEV1PWycOuQ HTTP/1.1" 200 None
This is the scrape input form
This is the scrape input form
Starting new HTTPS connection (1): www.coindesk.com:443
https://www.coindesk.com:443 "GET /business/2023/06/22/opnx-files-defamation-lawsuit-against-mike-dudas-issues-justice-tokens/?utm_medium=social&utm_term=organic&utm_campaign=coindesk_main&utm_content=editorial&utm_source=twitter HTTP/1.1" 200 None
This is the scrape input form
